integration/hostuser_test.go

```diff
@@ -583,6 +583,35 @@ func TestRootHostUsers(t *testing.T) {
 		require.NoError(t, err)
 		require.False(t, hasExpirations)
 	})
+
+	t.Run("Test migrate unmanaged user", func(t *testing.T) {
+		t.Cleanup(func() { cleanupUsersAndGroups([]string{testuser}, []string{types.TeleportKeepGroup}) })
+
+		users := srv.NewHostUsers(context.Background(), presence, "host_uuid")
+		expectedHome := filepath.Join("/home", testuser)
```

Comment:
should we use a temp dir?

Reply:
We actually shouldn't be creating a directory at all, this was left over from the test I coped as a base for this one 😅 There's a few tests in here that rely on creating directories in `/home` that probably need to be updated, so I'll make a note to fix those in a separate PR once I'm finished with v17 testing :+1: